src/uwtools/logging.py

```diff
@@ -16,6 +17,27 @@
 MSGWIDTH = 69
 
 
+class _Logger:
+    """
+    Support for swappable loggers.
+    """
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger()  # default to Python root logger.
+
+    def __getattr__(self, attr: str) -> Any:
+        """
+        Delegate attribute access to the currently-used logger.
+
+        :param attr: The attribute to access.
+        :returns: The requested attribute.
+        """
+        return getattr(self.logger, attr)
+
+
+log = _Logger()
```

Comment:
This is the singleton `_Logger` object to be used throughout the lifetime of a `uwtools` CLI invocation or API session.